roles/falcon_installation/tasks/api.yml

```
@@ -47,17 +47,28 @@
       msg: "No Falcon Sensor Update Policy with name: {{ falcon_sensor_update_policy_name }} was found!"
     when: falcon_sensor_update_policy_info.json.resources[0] is not defined
 
+  - name: "CrowdStrike Falcon | Validate Sensor Update Policy request for aarch64 architectures"
+    ansible.builtin.fail:
+      msg: "No Falcon Sensor Update Policy with name: {{ falcon_sensor_update_policy_name }} and enabled for aarch64 was found!"
+    when: falcon_sensor_update_policy_info.json.resources[0].settings.variants[0] is not defined and ansible_machine == "aarch64"
```

```suggestion
    when: 
      -  falcon_sensor_update_policy_info.json.resources[0].settings.variants[0] is not defined
      -  ansible_machine == "aarch64"
```
Use this format when using multiple AND statements. Easier to read and keeps length of lines lower.